README.md

```diff
@@ -9,12 +9,14 @@ like `bankai`.
 ## Usage
 ```js
 var choo = require('choo')
+var sw = require('choo-service-worker')
+var clear = require('choo-service-worker/clear')
 
 var app = choo()
 if (process.env.NODE_ENV !== 'production') {
-  app.use(require('choo-service-worker/clear')())
+  app.use(clear())
 }
-app.use(require('choo-service-worker')())
+app.use(sw())
```

Comment:
I change that part, because if you want to use some of the events or state new properties is better to have a single require statement.
Anyway, you are right that for that minimal example and use is better a conditional require, so I fix it and add a quote pointing to the complete example.